.buildkite/pipeline.gpu_large.yml

```diff
@@ -49,3 +49,16 @@
     - pip install -Ur ./python/requirements/ml/requirements_ml_docker.txt
     - ./ci/env/env_info.sh
     - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=gpu,-timeseries_libs,-py37,-post_wheel_build doc/...
+
+- label: ":tv: :python: Lightning 2.0 Train GPU tests"
```

Comment:
Can we update the emojis 😄  https://github.com/buildkite/emojis

:zap:

Reply:
Good idea!!